### ISSUE TITLE
Unable to activate challenge in Direct bcs of Not a Number error

### ISSUE THREAD

**Issue by JayaDhaka** [appirio-tech/direct-app#447]
Received NaN error in prize amount while activating this challenge https://www.topcoder.com/direct/contest/detail.action?projectId=30067498. Please check and fix the issue.

Here is the error screenshot:

https://user-images.githubusercontent.com/34749364/42362238-feff1788-810f-11e8-998a-c9de90f36f27.png

@skyhit Please fix this.

**Comment by skyhit** [comment]
@ajefts I can not access, @JayaDhaka what price did you set?

**Comment by JayaDhaka** [comment]
@rootelement Can you please shed some light on this issue?

**Comment by rootelement** [comment]
The background is this:
1. Used https://payments.topcoder.com - The payment amount was $350
2. The payment in the payment tool hung with a 504 error
3. I went into direct, saw the challenge in the project as draft. Tried to activate it, but it said it had no description or tags. Added the two main description fields and the two tags fields, then clicked save/activate
4. I saw the pop-up above.
5. Submitted a ticket for support to push through the issue, which they did.

**Comment by skyhit** [comment]
@ajefts @JayaDhaka can you give me access to the challenge?

**Comment by JayaDhaka** [comment]
@skyhit you can access the challenge now.

**Comment by skyhit** [comment]
@JayaDhaka still can not, I am getting "Sorry, you don't have permission to access this contest. " page

**Comment by JayaDhaka** [comment]
@skyhit please check now.

**Comment by skyhit** [comment]
@JayaDhaka my handle is `FireIce`, not `skyhit`, maybe this is the problem

**Comment by JayaDhaka** [comment]
@skyhit Yes, you are right. I have added you to the project.

**Comment by skyhit** [comment]
@JayaDhaka looks like the challenge is already completed?

**Comment by rootelement** [comment]
Yes, it's not necessarily this challenge, but multiple people hit that same error, and i was the one to capture the actual screenshot, so it's some sort of larger issue between the payment tool, the api, and direct.

**Comment by cwdcwd** [comment]
@skyhit can you look through the logs for this specific challenge?

**Comment by skyhit** [comment]
@cwdcwd how can I access the logs? @rootelement are there detail steps to follow to reproduce it?

**Comment by rootelement** [comment]
Up there: https://github.com/appirio-tech/direct-app/issues/447#issuecomment-403470818

**Comment by cwdcwd** [comment]
@skyhit I've found that if you adjust the prize money, save, and then attempt activation it works. 
I'm going to guess that something is not being saved for prizes when the API creates the challenge as opposed to when Direct saves the prizes. Thoughts?

**Comment by cwdcwd** [comment]
also, it seemed to wipe out the assignee on activation

**Comment by skyhit** [comment]
@cwdcwd for adjust the prize, you mean do in direct-app?

**Comment by skyhit** [comment]
@JayaDhaka @cwdcwd I will add debug code for problem reproduce.

**Comment by cwdcwd** [comment]
@skyhit just circling back on this one. anything show up in the logs at all?
